R/MzTabReader.R

```diff
@@ -189,6 +201,19 @@ getEvidence = function()
 
   
   renameColumns(res, name)
+  
+  scan_name = list("scan.event.number")
+  names(scan_name) = grepv("opt.global.ScanEventNumber|opt.global.cv.MS.1000776.scan.number.only.nativeID.format", colnames(res))
```

Comment:
while this works, this could be achieved with less code by simply
```
...
opt.global.ScanEventNumber = "scan.event.number",
opt.global.cv.MS.1000776.scan.number.only.nativeID.format = "scan.event.number"
...
```
in line 194 and then simply call `renameColumns(...)`?
A bit more fancy would be to check for duplicate values within renameColumns and try all variants, while only reporting a warning if all of them fail (exactly one should work)